tests/unit/amo/reducers/testReviews.js

```diff
@@ -891,5 +892,81 @@ describe(__filename, () => {
       const state = addReviewToState({ state: prevState, review });
       expect(state.byUserId[userId]).not.toBeDefined();
     });
+
+    it('resets ratingSummary when adding a new rating', () => {
```

Comment:
It seems like it should but just double checking that this will cover updating a rating too?

Reply:
Ah, yes, I forgot that. Thanks.